### PR TITLE
Create packaging-deploying.adoc

### PR DESCRIPTION
**Pull request by Charlotte-Holt**

### CODE REVIEW

File: ref/general/packaging-deploying.adoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+:seo-description: You can choose how to package and deploy your application with Open Liberty to meet your needs.
+:page-layout: general-reference
+:page-type: general
+= Package and deploy applications with Open Liberty
+
+You can choose how to package and deploy your application with Open Liberty to meet your needs.
+
+== Package, containerize, and deploy using Maven and Docker
+
+One of the easiest ways to get your cloud-native application up and running is to use Maven to package it with the Open Liberty runtime and use Docker to containerize and deploy it.
```
Comment (dmuelle) [inline ref/general/packaging-deploying.adoc]: you might move the action out of "use" and into the more specific verbs- something like

One of the easiest ways to get your cloud-native application up and running is to package it with Maven and containerize and deploy it with Docker .

maybe add a sentence or two about why you'd want to package the app with the server and deploy in a container? What's the advantage? Does it make the application more portable, lightweight?

Also- explain Maven before you introduce the plug-in

File: ref/general/packaging-deploying.adoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+The Open Liberty Maven plug-in includes goals that allow you to manage your Open Liberty server and applications.
+Go to the link:https://github.com/OpenLiberty/ci.maven[documentation] for information about how to enable the Open Liberty Maven plug-in for your project.
+Use the latest Open Liberty Docker images for access to the most recent feature versions and link:/docs/ref/general/#security-vulnerabilities.html[security fixes].
+You can download Open Liberty Docker images from link:https://hub.docker.com/_/open-liberty[Docker Hub].
+
+Maven is a build automation tool that makes managing your builds and dependencies easier.
```
Comment (dmuelle) [inline ref/general/packaging-deploying.adoc]: move this to the opening paragraph?

File: ref/general/packaging-deploying.adoc [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+To learn more about using Maven with Open Liberty, check out this guide on link:/guides/maven-intro.html[Building a web application with Maven].
+
+When you're ready to package your application, run the `mvn package` command.
+This command compiles and builds the project.
+Then, you can use the `docker build` and `docker run` commands to build your image and run it as a container.
+To learn more about running your applications with the Open Liberty Docker images, see this guide on link:/guides/containerize.html[Containerizing microservices].
+
+== Package into a JAR file using Maven
+
+In some cases, you might want to package you application into a JAR file.
```
Comment (dmuelle) [inline ref/general/packaging-deploying.adoc]: what are these cases? why would you want to? I think there's some mention of it in the pack and deploy guide

File: ref/general/packaging-deploying.adoc [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Run the following Maven goal to specify runnable JAR file as the packaging type:
+
+----
+mvn liberty:package -Dinclude=runnable
+----
+
+This command overrides the `include` property to package your application and runtime into a runnable JAR file.
+
+== Package using the server command
+
```
Comment (dmuelle) [inline ref/general/packaging-deploying.adoc]: is this an alternative to using Maven? Any benefits to doing it direct from the CLI?

File: ref/general/packaging-deploying.adoc [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+:page-type: general
+= Package and deploy applications with Open Liberty
+
+You can choose how to package and deploy your application with Open Liberty to meet your needs.
+
+== Package, containerize, and deploy using Maven and Docker
+
+One of the easiest ways to get your cloud-native application up and running is to use Maven to package it with the Open Liberty runtime and use Docker to containerize and deploy it.
+The Open Liberty Maven plug-in includes goals that allow you to manage your Open Liberty server and applications.
+Go to the link:https://github.com/OpenLiberty/ci.maven[documentation] for information about how to enable the Open Liberty Maven plug-in for your project.
```
Comment (dmuelle) [inline ref/general/packaging-deploying.adoc]: maybe move the plug-in info to a new paragraph and also mention dev mode & link to that topic. It's a goal of the maven plugin

File: ref/general/packaging-deploying.adoc [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+----
+mvn liberty:package -Dinclude=runnable
+----
+
+This command overrides the `include` property to package your application and runtime into a runnable JAR file.
+
+== Package using the server command
+
+You can use the `server package` command to package your application.
```
Comment (dmuelle) [inline ref/general/packaging-deploying.adoc]: Do you want to discuss gradle?

File: ref/general/packaging-deploying.adoc [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+The Open Liberty Maven plug-in includes goals that allow you to manage your Open Liberty server and applications.
+Go to the link:https://github.com/OpenLiberty/ci.maven[documentation] for information about how to enable the Open Liberty Maven plug-in for your project.
+Use the latest Open Liberty Docker images for access to the most recent feature versions and link:/docs/ref/general/#security-vulnerabilities.html[security fixes].
+You can download Open Liberty Docker images from link:https://hub.docker.com/_/open-liberty[Docker Hub].
+
+Maven is a build automation tool that makes managing your builds and dependencies easier.
+You use a Project Object Model (POM) file to define information about your project, and Maven downloads and installs all dependencies for you.
+To learn more about using Maven with Open Liberty, check out this guide on link:/guides/maven-intro.html[Building a web application with Maven].
+
+When you're ready to package your application, run the `mvn package` command.
```
Comment (dmuelle) [inline ref/general/packaging-deploying.adoc]: need the maven plugin enabled. Should this example be on it's own line?

File: ref/general/packaging-deploying.adoc (hunk 3 above)
Comment (dmuelle) [reply ref/general/packaging-deploying.adoc]: Also- change `you application` to `your application`

File: ref/general/packaging-deploying.adoc [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+== Package
+Before you containerize and deploy your application, package it with the Open Liberty runtime.
+There are different tools that you can use to package your application.
+Using Maven or Gradle to build and package your application is one of the easiest ways to get your application ready for cloud deployment.
+Maven and Gradle both work well as part of CI/CD pipelines.
+They start with the Open Liberty kernel and download specific features versions that are listed as dependencies.
+
+=== Maven
+link:https://maven.apache.org/[Maven] is a build automation tool that makes managing your builds and dependencies easier.
+With Maven, you use a Project Object Model (POM) file written in XML to define information about your project, and Maven downloads and installs all dependencies for you.
```
[the remark this answers is not in the document]
Comment (Charlotte-Holt) [reply ref/general/packaging-deploying.adoc:28]: I'm going to leave this as is for now - I just wanted to differentiate it between Gradle's DSL.

**Pull request merged**